docs/accounts/tenant/workspace-settings.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Workspaces
+sidebar_label: Workspaces
+---
+
+# Workspaces
+
+The **Workspaces** page allows you to control various settings which applies to all workspaces across the tenant.
+
+## Snapshot Visibility
+
+Snapshot Visibility setting allows you to control the permitted snapshot visibility options across all workspaces in your tenant.
```

Comment:
Missing article, and bold the term -  should be `The **Snapshot Visibility** setting...`

Reply:
Maybe add another clarifying sentence?  Something along the lines of "By default, workspace admins can share snapshots with anyone using a private link, but you may restrict snapshot sharing for your tenant". (is that accurate?)

Reply:
Done